README.md

```diff
@@ -1,3 +1,178 @@
 # school-library
 This is an application that have book records, adds the names of new teachers and students, add new books and save records of who borrowed a given book and when.
 
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [school-library](#school-library)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[your\_project\_name\] ](#-school-library-)
+  - [🛠 Built With PortsgreSQL](#-built-with-ruby)
+    - [Tech Stack Github](#tech-stack-github)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [school-library] <a name="about-project"></a>
+
+
+**[school-library]** This is an application that have book records, adds the names of new teachers and students, add new books and save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="SQL">Ruby, WSL</a>
+
+###<a name="tech-stack">Tech Used</a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Local Server</summary>
+  <ul>
+    <li><a href="https://ubuntu.com/">Ubuntu</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[module]** This contains all ruby files.
+- **[Class]** This contains all the methods.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a></a>
+
+- Project will not be deployed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a> 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+ git install postgreSQL
+ git install WSL
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-enumerable
+  git clone [git@github](https://github.com/ikoote1/school-library.git)
+
+### Install
+
+Install this project with:
+
+  cd Decode-morse-code
+  git install
+
+### Usage
+
+To run the project, execute the following command:
+
+  rails server
```

Comment:
[OPTIONAL] For your usage section,  I suggest you add the right command for making use of your project as this makes it easier for a user to work through your project👍. Kindly add the command used in triggering the entry point of the application👍.